packages/babel-plugin-jsx/test/index.test.tsx

```diff
@@ -13,7 +13,7 @@ const patchFlagExpect = (
   flag: number,
   dynamic: string[] | null,
 ) => {
-  const { patchFlag, dynamicProps } = wrapper.vm.$.subTree;
+  const { patchFlag, dynamicProps } = wrapper.vm.$.subTree as any;
```

Comment:
This was made internal in v3.2.0: https://github.com/vuejs/vue-next/commit/8610e1c9e23a4316f76fb35eebbab4ad48566fbf